lib/core/Head.js

```diff
@@ -150,10 +150,19 @@ class Head extends React.Component {
           })}
 
         {/* Site defined code. Keep these at the end to avoid overriding. */}
-        <link
-          rel="stylesheet"
-          href={this.props.config.baseUrl + 'css/main.css'}
-        />
+        {this.props.hash ? (
+          <link
+            rel="stylesheet"
+            href={
+              this.props.config.baseUrl + 'css/' + this.props.hash + '.main.css'
```

Comment:
Isn't it more common to have `main.[hash].css`? At least that's the webpack convention.

Reply:
Got it